x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/literal/Literals.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.sql.expression.literal;
+
+import org.elasticsearch.common.io.stream.NamedWriteableRegistry;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Utility class for common literal-related functions
+ */
+public final class Literals {
```

Comment:
I am not sure Literals is the right name for this class... or even the right package for it...

Reply:
That's what @costin suggested in https://github.com/elastic/elasticsearch/pull/39973#discussion_r265453767. Do you have other suggestions?

Reply:
Ok. No other suggestions. `Literals` it is :-).